### DISCUSSION TITLE
Import dataset

### DISCUSSION THREAD

**Discussion by zacharza** [posit-dev/positron#4919] [Feedback]
Very nice improvements. I am just missing gui for import dataset function that was part of R Studio. It was very comfortable to apply it for importing excel sheet via visual interface.
![import dataset](https://github.com/user-attachments/assets/648c0348-e11a-4b23-8437-b20b4d12c57f)

**Comment by juliasilge** [comment]
Thanks for this feedback! We have talked about possibly adding a "data import wizard", much like RStudio's. We don't have this work slated for the very near future but it is helpful for us to hear what people are missing from other IDEs they have used. 🙌

**Reply by Stonepeople** [reply]
I'd like to second this. I am loving Positron - but I had the long term habit in RStudio of keeping my data in a folder separately from my scripts and plots. I am lazy/bad at typing paths, so I used to use the import dataset GUI to generate the path to my data to use in my read_csv(). 
In Positron's data explorer I can, of course, use "open file" - but this means leaving the folder I am in. Or I can click on a csv - which opens up nicely as a preview, but doesn't generate a trail to it. A right-click would be lovely at this point (when someone has time to create that option). Thanks - and thanks for a super new IDE

**Comment by jonvanausdeln** [comment]
There is now an issue to track this feature request: https://github.com/posit-dev/positron/issues/5515
